src/js/es5.js

```diff
@@ -2,7 +2,7 @@ import './config/staging.js';
 import '@webcomponents/webcomponentsjs';
 import 'whatwg-fetch';
 import './polyfills/endswith.js';
-import '@cagov/accordion';
+// import '@cagov/accordion'; accordion indicator svgs are broken, disable accordions in IE until fixed
```

Comment:
currently in production all accordions are inactive in IE because of many js errors preventing execution, luckily we use progressive enhancement so everything is in an expanded state, commenting this out does not change the state of production systems but we should fix the svg indicator because accordions did work nicely in IE